main/php.h

```diff
@@ -295,7 +295,7 @@ ssize_t pread(int, void *, size_t, off64_t);
 #endif
 
 BEGIN_EXTERN_C()
-void phperror(char *error);
```

Comment:
Is this related?

Reply:
I don't think it was, must've sneaked in. I'll revert.